trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.controller.js

```diff
@@ -77,27 +97,27 @@ const postAmendCoverEndDate = async (req, res) => {
     });
   }
 
+  const baseUrl = `/case/${dealId}/facility/${facilityId}/amendment/${amendmentId}`;
+  const fallbackUrl = `/case/${dealId}/facility/${facilityId}#amendments`;
+
   try {
     let formatCurrentCoverEndDate = currentCoverEndDate;
     // convert the current end date to EPOCH format
     formatCurrentCoverEndDate = getUnixTime(new Date(formatCurrentCoverEndDate).setHours(2, 2, 2, 2));
+
     const payload = { coverEndDate, currentCoverEndDate: formatCurrentCoverEndDate };
-    const { status } = await api.updateAmendment(facilityId, amendmentId, payload, userToken);
 
-    if (status === 200) {
-      if (isTfmFacilityEndDateFeatureFlagEnabled()) {
-        return res.redirect(`/case/${amendment.dealId}/facility/${facilityId}/amendment/${amendmentId}/is-using-facility-end-date`);
-      }
-      if (amendment.changeFacilityValue) {
-        return res.redirect(`/case/${amendment.dealId}/facility/${facilityId}/amendment/${amendmentId}/facility-value`);
-      }
-      return res.redirect(`/case/${amendment.dealId}/facility/${facilityId}/amendment/${amendmentId}/check-answers`);
+    const { status } = await api.updateAmendment(facilityId, amendmentId, payload, userToken);
+    if (status !== HttpStatusCode.Ok) {
+      throw new Error('Error updating amendment with cover end date');
```

Comment:
could add function name to this error so we know where the error occurred.  Also having a console.error would be good

Reply:
We get the error logged in the catch, and the funciton name will be in the stack trace (along with LoC etc)

Reply:
I don't think the name of the function (AKA the controller) will be logged out specifically - rather, the endpoint that's calling this controller will be logged out. This seems OK to me to be aligned with other routes/controllers.

If this was in an async or "non-async, but critical" helper function outside of the controller I would suggest adding the function name for improved future debugging. However raises a discussion/decision/TODO about aligning other such helper functions - would feel strange to have e.g 1x helper logging out the function name when no other helpers do that

Reply:
I think should all be aligned so will resolve for this PR